### DISCUSSION TITLE
learning methodology

### DISCUSSION THREAD

**Discussion by AVI-SHUKLA1110** [Cyfrin/foundry-full-course-cu#489] [Q&A]
hey everyone !  I'd love to hear your suggestions on the best ways to fully understand and grasp such a vast course material. How do you recommend approaching such an extensive course most effectively ?"

**Comment by 0xusmanf** [comment]
My 1 cent is to practice as much as you can along with the lessons.
don't lose hope if you get stuck somewhere, ask for help. We have an amazing community here, we will be happy to help.
Try minting the challenge NFTs.
Take your time and don't rush through the course.
